cmd/kubectl-gadget/capabilities.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/capabilities/types"
+	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
+	"github.com/spf13/cobra"
+)
+
+var capabilitiesCmd = &cobra.Command{
+	Use:   "capabilities",
+	Short: "Trace open() system calls",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// print header
+		switch params.OutputMode {
+		case utils.OutputModeCustomColumns:
+			fmt.Println(getCustomOpensnoopColsHeader(params.CustomColumns))
+		case utils.OutputModeColumns:
+			fmt.Printf("%-16s %-16s %-16s %-16s %-6s %-6s %-16s %-4s %-16s %-6s\n",
+				"NODE", "NAMESPACE", "POD", "CONTAINER",
+				"UID", "PID", "COMM", "CAP", "NAME", "AUDIT")
+		}
+
+		config := &utils.TraceConfig{
+			GadgetName:       "capabilities",
+			Operation:        "start",
+			TraceOutputMode:  "Stream",
+			TraceOutputState: "Started",
+			CommonFlags:      &params,
+		}
+
+		err := utils.RunTraceAndPrintStream(config, capabilitiesTransformLine)
+		if err != nil {
+			return fmt.Errorf("failed to run tracer: %w", err)
```

Comment:
In case https://github.com/kinvolk/inspektor-gadget/pull/539 is merged first. Please update the error management.